src/main/java/com/iridium/iridiumskyblock/listeners/EntityDamageListener.java

```diff
@@ -3,46 +3,173 @@
 import com.iridium.iridiumcore.utils.StringUtils;
 import com.iridium.iridiumskyblock.IridiumSkyblock;
 import com.iridium.iridiumskyblock.PermissionType;
+import com.iridium.iridiumskyblock.configs.Configuration;
 import com.iridium.iridiumskyblock.database.Island;
+import com.iridium.iridiumskyblock.database.IslandTrusted;
+import com.iridium.iridiumskyblock.database.User;
+import com.iridium.iridiumskyblock.managers.CooldownProvider;
 import org.bukkit.entity.Player;
 import org.bukkit.entity.Projectile;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityDamageEvent;
 
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
 
 public class EntityDamageListener implements Listener {
 
-    @EventHandler(ignoreCancelled = true, priority = EventPriority.HIGHEST)
+    private static final List<EntityDamageEvent.DamageCause> membersExcludedDamages = new ArrayList<>();
```

Comment:
Why are u storing this here? just get directly from config

Reply:
for 1.13-1.15, these version haven't some causes like `HOT_FLOOR` so i want parsing these instead disabling plugin

Reply:
You can just work with the String, tho? You don't have to parse the DamageCause

Reply:
i just want remind it, "u added this but isn't exist"

Reply:
You could do that without saving them in this list.

Reply:
How, we're working with 1.17 api how i can check its exist on olders
If i put these with Enums instead string
Plugin will disable due cannot parsed them

i guess you're saying, they can be stay as string.
But check them in loadConfigs method and just throw a "cannot parsed" warn
Right ?